apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4PatientResourceProvider.java

```diff
@@ -906,22 +907,31 @@ private Patient queryDatabaseByHash(
           benesByHashOrIdQueryNanoSeconds,
           matchingBenes.size());
     }
-
     // Then, if we found more than one distinct BENE_ID, or none, throw an error.
     long distinctBeneIds =
         matchingBenes.stream()
             .map(Beneficiary::getBeneficiaryId)
             .filter(Objects::nonNull)
             .distinct()
             .count();
+    List<Long> distinctBeneIdsList =
```

Comment:
couldn't we move this down to where we've determined that the distinctBeneIds > 1 ?